Classes/Command/InstallCommandController.php

```diff
@@ -149,10 +152,20 @@ public function fixFolderStructureCommand()
                 $this->outputLine($fixedStatusObject->getTitle());
             }
         }
+
+        /** @var $installUtility \TYPO3\CMS\Extensionmanager\Utility\InstallUtility */
+        $installUtility = $this->objectManager->get(\TYPO3\CMS\Extensionmanager\Utility\InstallUtility::class);
```

Comment:
as pointed out below, we can't simply do it like this unfortunately. 

My idea is to gather the information (folders to be created) from all active extensions and put this info into the array of to be created folders and let the folder structure facade create the folders